plugin/checkout_handlers.go

```diff
@@ -0,0 +1,100 @@
+package plugin
```

Comment:
Not a request for this PR, but this project might be a good opportunity to restructure this repo layout to match the normal pattern (top level files + cmd folders)

Reply:
Yes, it might be. It would require a mirroring update in Vault to change which package is imported. I can do that as a bonus at the end of the project if I have spare time. At the moment, there are a couple of AWS tickets we'd like to sneak in if I get this nailed down early.